src/explore-education-statistics-admin/src/pages/release/content/components/KeyStatistics.tsx

```diff
@@ -137,44 +131,93 @@ const KeyStatistics = ({ release, isEditing }: KeyStatisticsProps) => {
 };
 
 const AddKeyStatistics = ({ release }: KeyStatisticsProps) => {
```

Comment:
This seems like a fairly large component to have in the same file, maybe better in it's own one?